modules/reference/pages/list-of-privileges.adoc

```diff
@@ -139,7 +139,6 @@ a|* `create group <pgName> proxy <rule>`
 
 |`CLEAR_GRAPHSTORE` |`clear graph store` |Yes
 
-|`ACCESS_TAG`
 a|
```

Comment:
```suggestion
|`ACCESS_TAG`

[Dropped in 4.1]
a|
```
If you wanted to remove this row from the table, you need to remove the entire row, not just this cell. However, since this page is now entitled "Legacy Privilege Syntax", it makes sense to continue to mention ACCESS_TAG.